deploy/operator-install.yaml

```diff
@@ -0,0 +1,742 @@
+#####################################################################
+### THIS IS AUTOGENERATED INFINISPAN OPERATOR INSTALL BUNDLE FILE ###
+###                  PLEASE DON'T EDIT THIS FILE                  ###
+#####################################################################
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: infinispan-operator
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  creationTimestamp: null
+  name: infinispan-operator
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - services
+  - services/finalizers
+  - endpoints
+  - persistentvolumeclaims
+  - events
+  - configmaps
+  - secrets
+  verbs:
+  - '*'
+- apiGroups:
+  - ""
+  resources:
+  - pods/exec
+  verbs:
+  - create
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  verbs:
+  - get
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  - deployments/finalizers
+  - daemonsets
+  - replicasets
+  - statefulsets
+  - statefulsets/finalizers
+  verbs:
+  - '*'
+- apiGroups:
+  - route.openshift.io
+  resources:
+  - routes
+  - routes/custom-host
+  verbs:
+  - '*'
+- apiGroups:
+  - networking.k8s.io
+  resources:
+  - ingresses
+  verbs:
+  - '*'
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - servicemonitors
+  verbs:
+  - get
+  - create
+- apiGroups:
+  - infinispan.org
+  resources:
+  - '*'
+  - caches
+  verbs:
+  - '*'
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: infinispan-operator
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  - nodes/status
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  - customresourcedefinitions/status
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - nodes/metrics
```

Comment:
@rigazilla can we join this resources with upper `nodes` section? Do we really need `watch` verb? I didn't see any watch subscriptions.

Reply:
@rigazilla could you please to look at the ClusterRole to optimize some permissions and compact file as small as possible.

Reply:
@dmvolod, I've merged the `nodes` section and removed watch

Reply:
Thanks, @rigazilla